lnmuxrpc/lnmux.proto

```diff
@@ -19,6 +19,10 @@ service Service {
     // Cancels an accepted invoice. In case settle has been requested
     // for an invoice, CancelInvoice returns a FailedPrecondition error.
     rpc CancelInvoice (CancelInvoiceRequest) returns (CancelInvoiceResponse);
+
+    // Lists settled or to be settled invoices. It returns invoices with a sequence number between
+    // 'index_offset' and 'index_offset + num_max_invoices'.
```

Comment:
Update comment.

Reply:
Sorry, Should have re-read more carefully

Reply:
It still says `index_offset` 😛

Reply:
Correction: I WILL re-read my PR 😄